### DISCUSSION TITLE
Password protection for remote access

### DISCUSSION THREAD

**Discussion by JackSpagnoli** [haugene/docker-transmission-openvpn#2092] [Q&A] [answered]
Hi,
I'm using this image on a remote server through a duckdns url and caddy to catch the request and reverse proxy to the container. The only glaring issue is that since there's no password protection on the webui I'm effectively just letting anyone download things onto my server. 

Ideally there would be a qBittorrent-nox style login page upon connecting to the webui, but (maybe I'm just dumb) I can't see how to enable this is transmission. What should I be doing?

**Comment by ilike2burnthing** [comment, marked as answer]
https://haugene.github.io/docker-transmission-openvpn/faq/#how_do_i_enable_authentication_in_the_web_ui

**Reply by JackSpagnoli** [reply]
Wow, I even read that FAQ page but somehow missed it. Thanks
